### ISSUE TITLE
Add new question on restricted data reuse in UA Ruhr catalog

### ISSUE THREAD

**Issue by ClaraRosalie** [FDM-UARuhr/rdmo-catalog-uaruhr#116]
**Describe the request**  
The following question is part of the Horizon2020 + ERC templates, but missing in RDMO:

> 2.2. Making data openly accessible: "If there are restrictions on use, how will access be provided?"

A new question should be created and added to the UA Ruhr and UA Ruhr Poposal catalog.

**Propose what exactly should be changed**  
Create new question as proposed below.

**Original setting or content**   
Similar questions in the catalog but none of them addresses this specific issue.

**Proposed setting or content**   
- URI:
- Key: restricted_access
- Attribute: Create new attribute: project/dataset/sharing/restricted_access
- Questionset: ua_ruhr/data-usage/data-sharing-and-re-use-publication
- Text (English): If there are restrictions on making the data openly accessible, how will restricted access be provided?
- Help (English): If data cannot be made publicly accessible, you can choose other options, e.g. grant access to a certain user group or to individuals on request only.
- Text (German): Falls die Daten nicht offen bereitgestellt werden können, wie wird der eingeschränkte Zugriff ermöglicht?
- Help (German): Können Daten nicht offen bereitgestellt werden, gibt es dennoch die Möglichkeit, den Zugriff zum Beispiel auf einen bestimmten Personenkreis einzuschränken oder nur für Einzelpersonen auf Anfrage zu ermöglichen.
- Widget Type: Textarea

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  
- [ ] uri
- [ ] key
- [ ] path
- [ ] questionset
  - [ ] position in set
- [ ] question text (en and ger)
- [ ] help text (en and ger)
- [ ] answer
  - [ ] widget type
  - [ ] value type
  - [ ] max/min, step and unit
- [ ] optionset
- [ ] condition
- [ ] attribute
